services/core-web/src/components/Forms/permits/GeneratePermitForm.js

```diff
@@ -220,6 +220,7 @@ export const GeneratePermitForm = (props) => (
         <FinalPermitDocuments
           mineGuid={props.noticeOfWork.mine_guid}
           noticeOfWork={props.noticeOfWork}
+          importNowSubmissionDocumentsJob={props.importNowSubmissionDocumentsJob}
         />
```

Comment:
Looks like FinalPermitDocuments is a connected component, can you get these props through the store vs passing to this component?